cypress/integration/parallel-2/article.elements.spec.js

```diff
@@ -109,19 +109,30 @@ describe('Elements', function () {
 		// 	getIframeBody().contains('View More on Instagram');
 		// });
 
-		it('should render the embed', function () {
+		it('should render the click to view overlay revealing the embed when clicked', function () {
 			const getIframeBody = () => {
 				return cy
 					.get('div[data-cy="embed-block"] > div > iframe')
 					.its('0.contentDocument.body')
 					.should('not.be.empty')
 					.then(cy.wrap);
 			};
+			const getClickToViewOverlay = () => {
+				return cy
+					.get('div[data-cy="click-to-view"]')
+					.its('0')
+					.should('not.be.empty')
+					.then(cy.wrap);
+			};
```

Comment:
We only need this pattern for iframes (which are treated differently in Cypress). To just click an element you can simply use `cy.get('button[data-cy=click-to-view-button]').click()` and the contains can just be `cy.contains('hosted on wnyc.org');` which checks the whole document.

Or you could (I'm fairly sure) chain them to restrict the contains like:

```
cy.get('button[data-cy=click-to-view-button]').contains('hosted on wnyc.org').click()
```